pandas/tests/indexes/multi/test_format.py

```diff
@@ -126,3 +92,251 @@ def test_repr_max_seq_item_setting(idx):
     with pd.option_context("display.max_seq_items", None):
         repr(idx)
         assert '...' not in str(idx)
+
+
+@pytest.mark.skipif(PY2, reason="repr output is different for python2")
```

Comment:
@topper-123 : Is it possible to use parameterization on this?  If you're just testing the same index for different output, you could condense into one test that checks different output each time (based on `PY2` flag)

Reply:
I guess it's *possible*, but the parametrizations would be removed in max 3½ months anyway and the parameters would be a bit unwieldy, because those strings are a bit large. So it will probably be a bit of work, and I'm not sure it is worth the effort for something that is on the way out.

As it is structured now, it will be very easy to remove the python2 specific tests, when the day comes. if needed, I could take it on me to remember to do it, when the time comes.

Reply:
@topper-123 : Did you address this?  I don't see any change here.

Reply:
I made it a bit clearer now...